src/components/NavBar/NavBar.js

```diff
@@ -2,7 +2,8 @@ import React, { useState } from 'react';
 import 'antd/dist/antd.css';
 import { Link, NavLink } from 'react-router-dom';
 
-import logo from '../../assets/blue-witness1.png';
```

Comment:
This is good to swap this out for clarity in labeling which logo we are using. For my sake, please go through and doublecheck that all occurences of 'logo' are swapped out for 'bwlogo'.

Reply:
my bad, getting on it right now